From 0a4bc46c34f662e655b7b2a9d956cf3b8a53559d Mon Sep 17 00:00:00 2001
From: Nickey Yang <nickey.yang@rock-chips.com>
Date: Mon, 24 Dec 2018 17:16:49 +0800
Subject: [PATCH] ARM: config: rockchip_linux: add dw-hdmi cec support.

this enables dw-hdmi cec support.

tested on rk3288-debian.
  apt-get install v4l-utils
  cec-ctl --playback -o Rockchip -V 0xaabbcc -M -T --cec-version-1.4

Change-Id: I2fb991410c96fe6f9a0d8a9739d89cc3084c23e8
Signed-off-by: Nickey Yang <nickey.yang@rock-chips.com>
---
 arch/arm/configs/rockchip_linux_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/configs/rockchip_linux_defconfig b/arch/arm/configs/rockchip_linux_defconfig
index 8587317d456b..d231e078a99c 100644
--- a/arch/arm/configs/rockchip_linux_defconfig
+++ b/arch/arm/configs/rockchip_linux_defconfig
@@ -289,6 +289,7 @@ CONFIG_MEDIA_SUPPORT=y
 CONFIG_MEDIA_CAMERA_SUPPORT=y
 CONFIG_MEDIA_CONTROLLER=y
 CONFIG_VIDEO_V4L2_SUBDEV_API=y
+CONFIG_MEDIA_CEC_SUPPORT=y
 CONFIG_MEDIA_USB_SUPPORT=y
 CONFIG_USB_VIDEO_CLASS=y
 # CONFIG_USB_VIDEO_CLASS_INPUT_EVDEV is not set
@@ -309,6 +310,7 @@ CONFIG_VIDEO_OV5647=y
 CONFIG_VIDEO_OV13850=y
 CONFIG_DRM=y
 CONFIG_DRM_ROCKCHIP=y
+CONFIG_DRM_DW_HDMI_CEC=y
 CONFIG_ROCKCHIP_DW_HDMI=y
 CONFIG_ROCKCHIP_DW_MIPI_DSI=y
 CONFIG_ROCKCHIP_ANALOGIX_DP=y
-- 
2.35.3

